packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -927,10 +1038,17 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
     const itemName = fileMetadata?.itemName ?? NotebookPanel.DEFAULT_NAME;
     const isMarkdown = itemName.endsWith('.md');
     const isExistingItem = fileMetadata?.id != null;
-    const overflowActions = this.getOverflowActions();
+    const minimap = {
+      minimap: { enabled: defaultNotebookSettings.isMinimapEnabled },
+    };
     const settings = {
       ...initialSettings,
+      ...minimap,
     };
```

Comment:
Doesn't look like it's from your change, but we should be memoizing this result (I'm not sure why we were even copying it before??), similar to how we're assigning `overflowActions`
Right now we're creating a new `settings` object on each re-render, which may give the wrong impression (something changing when it hasn't actually).